src/blocks/DataFrame_block.ts

```diff
@@ -21,7 +21,7 @@ export function df_create(headers: string[], data: any[][]): DataFrame {
 export function df_load_file(name: string): DataFrame {
     const csv = globalThis.uploadedDatasets[name];
```

Comment:
I think this may need to be updated to work with the new dataset uploading system